scripts/get-all-commands.ts

```diff
@@ -0,0 +1,53 @@
+import { exec } from 'child_process';
+import SpinniesManager from '../lib/ui/SpinniesManager';
+import util from 'util';
+
+const output = {};
+
+const execAsync = util.promisify(exec);
+
+async function extractCommands(
```

Comment:
This recurses over the commands running `--get-yargs-completions` and generates a JSON object to keep the hierarchy.  We can use this in the future to scrape help documents etc to auto generate docs from the help text in the CLI if we so choose.